tests/zstreamtest.c

```diff
@@ -1834,6 +1834,97 @@ static int basicUnitTests(U32 seed, double compressibility, int bigTests)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    DISPLAYLEVEL(3, "test%3i : External matchfinder API: ", testNb++);
+    {
+        size_t const dstBufSize = ZSTD_compressBound(CNBufferSize);
+        BYTE* dstBuf = (BYTE*)malloc(ZSTD_compressBound(dstBufSize));
```

Comment:
very minor :  
`BYTE* const dstBuf = `
This statement guarantees that `dstBuf` (as a pointer value) will not be updated after its initialization.